packages/core/src/invoke.ts

```diff
@@ -38,9 +38,10 @@ export function spawnMachine<
       ) as TMachine;
     }
     const childService = interpret(resolvedMachine, {
-      ...options, // inherit options from this interpreter
```

Comment:
this comment was not true, this is just spreading spawn options, not inheriting from the parent